src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -47,6 +62,20 @@ export class RundownComponent implements OnInit, OnDestroy {
     this.startOffsetsInMsFromPlayheadForSegments = this.getStartOffsetsInMsFromPlayheadForSegments(rundownTimingContext)
   }
 
+  private onActionsChanged(actions: Action[]): void {
+    this.videoClipActions = actions.filter((action): action is Tv2VideoClipAction => {
+      return this.entityParser.parseTv2Action(action) && (<Tv2Action>action).metadata?.contentType === Tv2ActionContentType.VIDEO_CLIP
```

Comment:
The entityParser are supposed to be used when we receive the data from the backend. They are not supposed to be used in the components. The data should already have been through the parser before it ever sees a component.

What do you need it for here?